spring-rabbit/src/test/java/org/springframework/amqp/rabbit/config/ListenerContainerParserTests.java

```diff
@@ -225,14 +225,22 @@ public void testAnonParent() throws Exception {
 	public void testIncompatibleTxAtts() {
 		try {
 			new ClassPathXmlApplicationContext(getClass().getSimpleName() + "-fail-context.xml", getClass()).close();
-			fail("Parse exception exptected");
+			fail("Parse exception expected");
 		}
 		catch (BeanDefinitionParsingException e) {
 			assertTrue(e.getMessage().startsWith(
 				"Configuration problem: Listener Container - cannot set channel-transacted with acknowledge='NONE'"));
 		}
 	}
 
+	@Test
+	public void AMQP_724_potentialNPE() {
+		final DefaultListableBeanFactory bf = new DefaultListableBeanFactory();
+		final XmlBeanDefinitionReader reader = new XmlBeanDefinitionReader(bf);
+		reader.setValidating(false);
+		reader.loadBeanDefinitions(new ClassPathResource(getClass().getSimpleName() + "-context.xml", getClass()));
```

Comment:
That file contains
`<rabbit:listener-container connection-factory="connectionFactory" channel-transacted="true" transaction-size="5" >`
and if running without validation acknowledgeMode is null, but with validation is has default value from xsd.

Or a separate file here?